app/javascript/components/Header/BasicHeader.tsx

```diff
@@ -60,12 +62,12 @@ export const BasicHeader = ({ page, toggleMobileNav, mobileNavOpen, primaryHeade
                 <Title>{page.title}</Title>
               }
             />
-            <NavMenuButton
+            {showMobileMenu && (<NavMenuButton
```

Comment:
Is there a reason to wrap the `NavMenuButton` in parentheses?

Reply:
Good point, actually we don't need it. I have removed it, kindly re-review it.